app/views/admin/works/_combined_audio_derivatives.html.erb

```diff
@@ -38,16 +62,48 @@
             <i class="fa fa fa-thumbs-down pr-1" aria-hidden="true"></i>
             Combined audio derivatives exist for this oral history, but they are out of date.
           </p>
+          <% if model&.oral_history_content&.queued? %>
+            <p class="alert-info p-3">
+            A job to recreate them was added to the queue <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.started? %>
+            <p class="alert-info p-3">
+            A job to recreate them started <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
+          <% if model&.oral_history_content&.failed? %>
+            <p class="alert-danger p-3">
+            A job to recreate them failed <%=view.time_since_job_status_change %>.
+            </p>
+          <% end %>
```

Comment:
This is just repeating the status info that was already earlier in this file? Is there a reason we need it in two places?

Reply:
Fixed!